en-US/R.xml

```diff
@@ -118,7 +118,8 @@
 			<term>refer to</term>
 			 <listitem>
 				<para>
-					Do not use to indicate a reference (within a manual) or a cross-reference (to another manual or documentation source). Use "See".
+					Use to refer readers to another resource. 
+					<!-- Do not use to indicate a reference (within a manual) or a cross-reference (to another manual or documentation source). Use "Refer to". -->
```

Comment:
Is this line missing the word "see"? "Do not use "see" to indicate...

Reply:
Won't fix: this line is the previous entry, now commented out.